packages/wallet/contracts/pre-built-artifacts/ERC20AssetHolder.json

```diff
@@ -0,0 +1,4676 @@
+{
```

Comment:
The majority of this file is not needed: IIRC, we only need `contractName`, `abi`, and `bytecode`. If you run 
```
$ jq '{abi: .abi, bytecode: .bytecode, contractName: .contractName}' build/contracts/$CONTRACT.json  > tmp && mv tmp contracts/pre-built-contracts/Migrations.json
```
then this diff is more manageable, and the git repository does not balloon

Reply:
(Assuming that you've got `jq` installed)

Reply:
Oh, we'll eventually need the addresses from the `networks` property, but
- it's not necessary to get the development & test environments working
- we are testing out deploys with `etherlime`, which makes the management of prebuilt-artifacts saner

Reply:
Got it, thanks :)